lock-api/src/main/java/com/palantir/lock/v2/DefaultTimelockService.java

```diff
@@ -17,18 +17,25 @@
 package com.palantir.lock.v2;
 
 import java.util.Set;
+import java.util.UUID;
 
 import com.palantir.timestamp.TimestampRange;
 
 public final class DefaultTimelockService implements TimelockService {
     private final TimelockRpcClient delegate;
+    private final UUID clientId;
 
-    private DefaultTimelockService(TimelockRpcClient timelockRpcClient) {
+    private DefaultTimelockService(TimelockRpcClient timelockRpcClient, UUID clientId) {
         this.delegate = timelockRpcClient;
+        this.clientId = clientId;
     }
 
     public static TimelockService create(TimelockRpcClient timelockRpcClient) {
-        return new DefaultTimelockService(timelockRpcClient);
+        return new DefaultTimelockService(timelockRpcClient, UUID.randomUUID());
+    }
+
+    public static TimelockService create(TimelockRpcClient timelockRpcClient, UUID clientId) {
+        return new DefaultTimelockService(timelockRpcClient, clientId);
```

Comment:
This one is only used in the timelock integration tests (in fact when I first looked at it I wondered why it existed in the first place). I think if we change the testable timelock cluster to use an RPC client (see comment there) we can remove this method.